packages/repository/test/unit/repository-mixin/repository-mixin.test.ts

```diff
@@ -69,7 +72,7 @@ describe('RepositoryMixin', () => {
 
   class AppWithRepoMixin extends RepositoryMixin(Application) {}
 
-  class NoteRepo {
+  class NoteRepo implements Repository<any> {
```

Comment:
I think this should be `Repository<Note>`, or at least `Repository<Entity>`. Thoughts?